packages/jsii-rosetta/bin/jsii-rosetta.ts

```diff
@@ -0,0 +1,152 @@
+import fs = require('fs-extra');
+import yargs = require('yargs');
+import { TranslateResult, DEFAULT_TABLET_NAME, translateTypeScript } from '../lib';
+import { PythonVisitor } from '../lib/languages/python';
+import { VisualizeAstVisitor } from '../lib/languages/visualize';
+import { extractSnippets } from '../lib/commands/extract';
+import logging = require('../lib/logging');
+import path = require('path');
+import { readTablet as readTablet } from '../lib/commands/read';
+import { translateMarkdown } from '../lib/commands/convert';
+import { File, printDiagnostics, isErrorDiagnostic } from '../lib/util';
+
+async function main() {
+  const argv = yargs
+    .usage('$0 <cmd> [args]')
+    .option('verbose', {
+      alias: 'v',
+      type: 'boolean',
+      desc: 'Increase logging verbosity',
+      count: true,
+      default: 0
+    })
+    .command('snippet FILE', 'Translate a single snippet', command => command
+        .positional('file', { type: 'string', describe: 'The file to translate (leave out for stdin)' })
+        .option('python', { alias: 'p', boolean: true, description: 'Translate snippets to Python' })
+    , wrapHandler(async args => {
+      const result = translateTypeScript(
+        await makeFileSource(args.file || '-', 'stdin.ts'),
+        makeVisitor(args));
+      renderResult(result);
+    }))
+    .command('markdown FILE', 'Translate a MarkDown file', command => command
+        .positional('file', { type: 'string', describe: 'The file to translate (leave out for stdin)' })
+        .option('python', { alias: 'p', boolean: true, description: 'Translate snippets to Python' })
+    , wrapHandler(async args => {
+      const result = translateMarkdown(
+        await makeFileSource(args.file || '-', 'stdin.md'),
+        makeVisitor(args));
+      renderResult(result);
+    }))
+    .command('extract [ASSEMBLY..]', 'Extract code snippets from one or more assemblies into a language tablets', command => command
```

Comment:
Not really, the `md` and `ts` modes output directly to stdout, and they can also be used to visualize the AST instead of actually rendering it to a language.

They were used for testing while developing. I guess we could also get rid of them.